inst/extdata/parameters.csv

```diff
@@ -2,8 +2,7 @@ pathogen_id,type_id,study_id,year,size,distribution,mean,sd,quantile_025,median,
 adenovirus,incubation,Lessler_etal,2009,14,lnorm,NA,NA,NA,5.6,6.5,NA,NA,NA,NA,NA,NA,NA,1.72276435,0.22096732,yes,no,no,NA,19393959,10.1016/S1473-3099(09)70069-6,Adam Kucharski
 ebola,serial_interval,WHO_response_team,2014,92,gamma,15.3,9.3,NA,NA,NA,NA,NA,NA,NA,NA,2.706556,5.652941,NA,NA,yes,NA,NA,NA,25564903,10.1056/nejmoa1411100,Adam Kucharski
 ebola,incubation,WHO_team,2014,500,gamma,9.1,7.3,NA,NA,NA,NA,NA,NA,NA,NA,1.55395,5.856044,NA,NA,yes,NA,NA,Extracted using single exposures,25564903,10.1056/nejmoa1411100,Adam Kucharski
-ebola,onset_to_admission,WHO_team,2014,500,gamma,5,4.7,NA,NA,NA,NA,NA,NA,NA,NA,1.131734,4.418,NA,NA,yes,NA,NA,NA,25564903,10.1056/nejmoa1411100,Adam Kucharski
-ebola,onset_to_death,WHO_team,2014,500,gamma,7.5,6.8,NA,NA,NA,NA,NA,NA,NA,NA,1.216479,6.165333,NA,NA,yes,NA,NA,NA,25564903,10.1056/nejmoa1411100,Adam Kucharski

```

Comment:
Why have these values been removed?

Reply:
These parameter values don't align with distribution plots in original paper, even though mean/sd correct. Need to investigate further before reincluding.